mscore/preferences.cpp

```diff
@@ -313,7 +313,7 @@ void Preferences::write()
       s.setValue("myImagesPath", myImagesPath);
       s.setValue("myTemplatesPath", myTemplatesPath);
       s.setValue("myPluginsPath", myPluginsPath);
-      s.setValue("sfPath",  sfPath);
+      s.setValue("mySoundfontsPath", mySoundfontsPath);
```

Comment:
That will do for MuseScore 2.1 or 3.0 but not for 2.0.3. I will cherry pick the commit in 2.0.3 but change this back to sfPath or write compatibility code.